tools/on_device_tests_gateway.proto

```diff
@@ -52,6 +52,7 @@ message OnDeviceTestsCommand {
   string start_timeout = 20;
   string test_timeout = 21;
   string builder_url = 22;
+  string gcs_xml_result_path = 23;
```

Comment:
Still different naming here and in python script 😄

Reply:
Ugh, done